lib/ble/ble.toit

```diff
@@ -713,6 +727,69 @@ class DataBlock:
     result.replace 2 data
     return result
 
+  hash-code -> int:
+    return type.hash-code * 31 + data.hash-code
+
+  operator== other/any:
+    if other is not DataBlock: return false
+    other-block := other as DataBlock
+    return type == other-block.type and data == other-block.data
+
+  stringify --add-prefix/bool=true -> string:
+    prefix := add-prefix ? "DataBlock: " : ""
+    if type == TYPE-RAW: return "$(prefix)raw=$data"
+    if type == TYPE-FLAGS:
+      flags-string := ""
```

Comment:
I think I'd build this up as a list instead and then join.

Reply:
done.